container-images/mds-geography/index.ts

```diff
@@ -0,0 +1,21 @@
+/*
+    Copyright 2019 City of Los Angeles.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+ */
+
+import { ApiServer, HttpServer } from '@mds-core/mds-api-server'
+import { api } from '@mds-core/mds-geography'
+import { env } from '@container-images/env-inject'
+
+HttpServer(env().PORT ?? 4012, ApiServer(api))
```

Comment:
This is definitely out of scope for this PR, but in general I think I've seen a few different approaches to handling this env var/env vars in general, it would be great to establish a pattern company wide.

Reply:
This is the new, recommended way of doing it I believe.